tests/Versioning/VersionHistoryTest.php

```diff
@@ -311,6 +313,119 @@ public function testDeleteUnexistingVersion()
     }
 
     /**
+     * Load Version by label
+     */
+    public function testGetVersionByLabel()
+    {
+        $history = $this->vm->getVersionHistory('/tests_version_base/versioned');
+        $history->addVersionLabel('1.0', 'stable', false);
+
+        $expectedVersion = $history->getVersion('1.0');
+        $actualVersion = $history->getVersionByLabel('stable');
+
+        $this->assertEquals($expectedVersion->getIdentifier(), $actualVersion->getIdentifier());
+    }
+
+
+    /**
+     * Try to load  Version by unexisting label
+     * @expectedException PHPCR\Version\VersionException
```

Comment:
please use leading `\` to not confuse IDEs that think this could be a relative path.